scene/3d/ray_cast_3d.cpp

```diff
@@ -326,7 +421,8 @@ void RayCast3D::_create_debug_shape() {
 
 		Ref<StandardMaterial3D> line_material = static_cast<Ref<StandardMaterial3D>>(debug_material);
```

Comment:
Same here, you could replace the whole block with `_update_debug_shape_material();`